plugins/aws/importers.go

```diff
@@ -94,10 +97,27 @@ var backendNames = map[keyring.BackendType]string{
 	keyring.PassBackend:          "Pass",
 }
 
+func fileKeyringPassphrasePrompt(prompt string) (string, error) {
+	if password, ok := os.LookupEnv("AWS_VAULT_FILE_PASSPHRASE"); ok {
+		return password, nil
+	}
+
+	fmt.Fprintf(os.Stderr, "%s: ", prompt)
+	b, err := term.ReadPassword(int(os.Stdin.Fd()))
```

Comment:
I wasn't able to test this because I cannot get encrypted files to work correctly as a backend to my aws-vault.
<img width="982" alt="Screenshot 2023-04-26 at 15 39 49" src="https://user-images.githubusercontent.com/45081667/234593634-8e7b8058-fec3-427c-ab13-0130d3bfaa03.png">
Is this working as expected for you @williamhpark ?

Reply:
Hey @AndyTitu, the issue is that you need to specify `--backend file` when you run `aws exec` as well (I know kind of weird). So running `aws-vault exec file-profile --backend file -- aws sts get-caller-identity` should work for you. It works for me.